apis/clusters/v1beta1/kafkaconnect_types.go

```diff
@@ -118,8 +122,16 @@ type KafkaConnectSpec struct {
 
 // KafkaConnectStatus defines the observed state of KafkaConnect
 type KafkaConnectStatus struct {
-	ClusterStatus        `json:",inline"`
-	TargetKafkaClusterID string `json:"targetKafkaClusterId,omitempty"`
+	GenericStatus `json:",inline"`
+
+	DataCentres []*KafkaConnectDataCentreStatus `json:"dataCentres,omitempty"`
+}
+
+type KafkaConnectDataCentreStatus struct {
+	GenericDataCentreStatus `json:",inline"`
+
+	NumberOfNodes int     `json:"numberOfNodes,omitempty"`
```

Comment:
please try to make it generic

Reply:
I think we should count it on our end. Also, I'm considering moving this field into the 'Status,' which allows us to use the print column annotations (for convenient displaying of the total nodes number) and simplifies the calculation when dealing with multiple data centers.